### ISSUE TITLE
The favicon.ico link in the header is invalid: 404 not found

### ISSUE THREAD

**Issue by McPringle** [dukecon/dukecon_webhome#17]
On every page request, the browser tries to load a favicon because the favicon file is linked in the HTML header section. But the file linked there does not exist.

**Comment by sippsack** [comment]
You are right. We have to investigate. The favicon is loaded dynamically depending on the conference. But something seems to go wrong.
